packages/lintcstubs/lintcstubs.0.4.7/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+synopsis: "OCaml C stub static analyzer"
+description:
+  "Uses a generated C model for how OCaml C primitives can be called. Run a static analyzer to find incorrect API/macro usage that leads to race conditions."
+maintainer: ["Edwin Török <edwin.torok@cloud.com>"]
+authors: ["Edwin Török <edwin.torok@cloud.com>"]
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/edwintorok/lintcstubs"
+bug-reports: "https://github.com/edwintorok/lintcstubs/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "ocaml" {>= "4.13"}
+  "lintcstubs-gen" {= version}
+  "goblint" {>= "2.1.0"}
+  "goblint-cil"
+  "dune-compiledb"
+  "fpath"
+  "fmt" {>= "0.9.0"}
+  "sexplib" {>= "v0.15.0"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "--promote-install-files=false"
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+  ["dune" "install" "-p" name "--create-install-files" name]
+]
+dev-repo: "git+https://github.com/edwintorok/lintcstubs.git"
+pin-depends: [
+	["dune-compiledb.0.4.0" "git+https://github.com/edwintorok/dune-compiledb.git#0.4.0"]
+]
```

Comment:
AFAIK opam-repository doesn't allow `pin-depends`, so you'll have to release this package first.